cmd/server/assets/header.html

```diff
@@ -229,25 +234,27 @@ <h6 class="dropdown-header">Actions</h6>
 {{end}}
 
 {{define "flash"}}
-{{range $msg := .flash.Errors}}
-<div class="alert alert-danger" role="alert">
-  {{$msg}}
-</div>
-{{end}}
-{{range $msg := .flash.Warnings}}
-<div class="alert alert-warning" role="alert">
-  {{$msg}}
-</div>
-{{end}}
-{{range $msg := .flash.Alerts}}
-<div class="alert alert-success alert-dismissible fade show" role="alert">
-  {{$msg}}
-  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-    <span aria-hidden="true">&times;</span>
-  </button>
+<div id="alerts">
```

Comment:
I can see us accidentially conflicting with this in the future. Can we make this `alerts-container` or something more specific of an ID?